src/wazuh_testing/utils/sockets.py

```diff
@@ -27,3 +28,18 @@ def delete_sockets(path=None):
                 os.remove(item)
     except FileNotFoundError:
         pass
+
+def send_request_socket(query, socket_path=WAZUH_DB_SOCKET_PATH):
+    """Send queries request to socket in the argument.
+
+    Args:
+        query (str): query request command. For example `agent {agent.id} rootcheck delete`.
+        socket_path (str): by default use wdb socket.
+    Returns:
+        list: Query response data.
+    """
+
+    controller = SocketController(socket_path)
+    controller.send(query, size=True)
+    response = controller.receive(size=True)
```

Comment:
Missing close socket:
```suggestion
    response = controller.receive(size=True)
    controller.close()
```